src/domain/gpuInstance.ts

```diff
@@ -52,7 +52,7 @@ export class GpuInstanceManager extends InstanceManager<GpuInstance> {
 
   protected override parseMessagesFilter({ content }: any): boolean {
     if (content === undefined) return false
-
+    return true
```

Comment:
Was this change on purpose?
We are considering all kind of instances GPU instances